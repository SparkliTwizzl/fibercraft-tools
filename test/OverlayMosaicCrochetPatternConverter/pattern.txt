10sc
5sc,5dc
10dc
4sc,6dc
10sc
3dc,7sc
10dc
6sc,4dc
10sc
2dc,8sc
