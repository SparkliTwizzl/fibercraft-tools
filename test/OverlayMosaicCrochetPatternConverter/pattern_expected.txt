    1. A          bs, [1] 10sc, bs
    2. B          bs, [1] 5sc, [6] 5dc, bs
    3. A          bs, [1] 10dc, bs
    4. B          bs, [1] 4sc, [5] 6dc, bs
    5. A          bs, [1] 10sc, bs
    6. B          bs, [1] 3dc, [4] 7sc, bs
    7. A          bs, [1] 10dc, bs
    8. B          bs, [1] 6sc, [7] 4dc, bs
    9. A          bs, [1] 10sc, bs
   10. B          bs, [1] 2dc, [3] 8sc, bs
